src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy.mdx

```diff
@@ -136,3 +136,135 @@ The policies index lists policies in alphabetical order. To view or search for e
+
+    1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, then click **Alert policies.**
+    2. From the policy row, click the delete <Icon name="fe-trash-2"/>
+       icon.
+
+    Any notification channels associated with a deleted or disabled policy will remain. You must [delete notification channels](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/delete-alert-notification-channels) or [remove them](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/add-or-remove-policy-channels) individually as necessary.
+
+    In a deleted policy, [incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) and [event violations](/docs/alerts/new-relic-alerts-beta/reviewing-events/review-events-across-products) are provided as a read only resource.
+  </Collapser>
+
+  <Collapser
+    id="targets-policy"
+    title="Check for entities (targets) assigned to a policy"
+  >
+    To check whether [an entity (target) has any policies assigned](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/identify-entities-without-alert-policies):
+
+    1. Go to the product (APM, browser, etc.).
+    2. From the selected product's index, check whether the entity's color-coded health status is light green with an icon. ![icon-alerts-q-green.png](./images/icon-alerts-q-green.png "Alerts v3: alert icons")
+
+    <Callout variant="tip">
+      **Exception:** Infrastructure does not use this feature.
+    </Callout>
+  </Collapser>
+
+  <Collapser
+    id="incidents"
+    title="Change how threshold violations roll up into incidents"
+  >
+    To select how [violations to your thresholds](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/specify-when-new-relic-creates-incidents) are grouped into incident records (by policy, by condition, or by target and condition):
+
+    1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, click **Alert policies**, then **(select a policy)**.
+    2. Click <Icon name="fe-settings"/>
+       **Incident preference**.
+  </Collapser>
+
+  <Collapser
+    id="link-to-ai"
+    title="Link to Incident Intelligence"
+  >
+    If you use Applied Intelligence, you can connect your conditions to [Incident Intelligence](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#configure-source-nr-alerts).
+
+    1. From **[one.newrelic.com](https://one.newrelic.com)**, click **Alerts**.
+    2. On the left under **Incident Intelligence**, click **Sources** and then click **Alerts**.
+    3. Select the policies you want to connect to Applied Intelligence, and click **Connect**.
+
+    You can add additional alerts policies or remove policies you've already connected in **Sources > New Relic Alerts**.
+  </Collapser>
+</CollapserGroup>
```

Comment:
This set of collapsers is great! Something to consider: When a list has more than 7-9 items, it gets harder to skim. In this case I don't think it makes sense to alphabetize for easier skimming because it's already pretty much alpha order by function. Maybe later consider breaking it into a couple of collapser sets, one with an H3 intro about basic functions and another with an H3 intro about more enhanced setup?

Reply:
I'll keep this in mind for future!